js/ui/windowManager.js

```diff
@@ -1641,14 +1642,19 @@ const WindowManager = new Lang.Class({
                          });
         } else {
             let monitor = Main.layoutManager.monitors[actor.meta_window.get_monitor()];
+            if (!monitor) {
+                this._unminimizeWindowDone(shellwm, actor);
+                return;
+            }
```

Comment:
This strikes me as a little strange - `monitor` would only be `null` in cases where the window didn't have a valid monitor. Is this a common case?

Reply:
I believe it can also be null if you don't have any monitor connected at the time.